src/main/java/org/folio/innreach/domain/service/impl/ContributionServiceImpl.java

```diff
@@ -185,11 +184,6 @@ private JobResponse getJobResponse(UUID id) {
   public ContributionDTO createOngoingContribution(UUID centralServerId) {
     log.info("createOngoingContribution:: parameters centralServerId: {}", centralServerId);
     var contribution = createEmptyContribution(centralServerId);
```

Comment:
Maybe instead of creating the empt contribution it should create a row that has a count of 1 for the message that we are processing here just to make the stats "correct". Also probably want to rename this method `createEmptyContribution` to something like `createContribution`.

Reply:
@steveellis In the current code implementation, `createEmptyContribution ` method initialize the stats value to 0 and gets saved. there are methods which increment the count with the given value. Please refer below.

```
public void addRecordsProcessed(int itemsCount) {
      recordsProcessed += itemsCount;
    }
```

Let's consider below 2 different scenarios where the stats will be different.
1) Instance Update - In this case , only instance contribution can happen. 
records_total - 1, records_processed - 1 , records_contributed - 1, records_decontributed - 0
2) Item delete- In this case , item decontribution and instance contribution can happen. 
records_total - 2, records_processed - 2 , records_contributed - 1, records_decontributed - 1

So, If we initialize to 1, then the stats will be showing incorrect. 
If you agree with the above, I think it is better to keep the name of the method as createEmptyContribution as we always initialize the stats related column value as 0.